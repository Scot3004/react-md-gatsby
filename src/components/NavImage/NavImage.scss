.md-drawer {
  .md-toolbar {
    height: 110px;
    background: url("../../images/mosaic.jpg");
  }
}

.nav-image {
  background: rgba(0, 0, 0, 0.54);
  width: 100%;

  .md-text {
    color: white;
    font-weight: 600;
  }

  .md-text--secondary {
    color: white;
    font-weight: 400;
  }
}
