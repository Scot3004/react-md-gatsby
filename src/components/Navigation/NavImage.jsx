import React, { Component } from "react";
import {CardTitle} from "react-md";
import "./NavImage.scss";

class NavImage extends Component {
  render() {
    const { config } = this.props;
    return (
      <div className="nav-image" >
        <CardTitle
          avatar={<img
            role="presentation"
            src={config.userAvatar}
            alt={config.userName}
            style={{
              width: 72,
              height: 72,
              padding: 10
            }}
          />}
          title={config.userName}
          subtitle="Web developer"
          style={{color: "white"}}
        />
      </div>
    )
  }
}

export default NavImage;
