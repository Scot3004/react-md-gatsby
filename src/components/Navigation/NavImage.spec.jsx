import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NavImage from './NavImage';

it('renders correctly', () => {
  const config = {
    userAvatar: "userAvatar",
    userName: "userName",
  }
  const tree = shallow(<NavImage config={config} />);
  expect(toJson(tree)).toMatchSnapshot();
});
