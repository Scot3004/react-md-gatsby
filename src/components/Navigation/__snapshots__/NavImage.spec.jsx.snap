// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="nav-image"
>
  <CardTitle
    avatar={
      <img
        alt="userName"
        role="presentation"
        src="userAvatar"
        style={
          Object {
            "height": 72,
            "padding": 10,
            "width": 72,
          }
        }
      />
    }
    style={
      Object {
        "color": "white",
      }
    }
    subtitle="Web developer"
    title="userName"
  />
</div>
`;
